src/main/java/teammates/logic/api/Logic.java

```diff
@@ -301,6 +301,15 @@ public void putDocument(InstructorAttributes instructor) {
     }
 
     /**
+     * Batch creates or updates documents for the given Instructors.
+     *
+     * @see InstructorsLogic#putDocuments(List)
+     */
+    public void batchPutInstructorDocuments(List<InstructorAttributes> instructors) {
```

Comment:
Ohh I see! I guess what @boxin-yang said then - update the `*Logic` method names to match. Also, I suggest renaming `batchPut*Documents` -> `put*Documents`, as the `s` in `Documents` already implies batch.